doc/faq.rst

```diff
@@ -131,6 +131,20 @@ For encrypted mailing lists like `schleuder
 Help welcome.
 
 
+How can I participate in Autocrypt?
+-----------------------------------
+
+First of all it would be nice if you join the discussion on IRC.
+Autocrypt lives in #autocrypt on irc.freenode.net, if you have ideas,
+discuss them there first.
+
+Apart from that, anyone can open a pull request to the spec or to the
+website. Feel free to share your thoughts and skills!
```

Comment:
I'd continue with using "you" here like in the first paragraph, so something like "We appreciate if you open a pull request to the spec or to the website".